uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -2065,4 +2071,106 @@ public function testLeaveRequestIsDeletedValueCanNotBeSetWhenCreatingALeaveReque
     $leaveRequestRecord = LeaveRequest::findById($leaveRequest->id);
     $this->assertEquals(0, $leaveRequestRecord->is_deleted);
   }
+
+  public function testEmailGetsSentWhenLeaveRequestIsCreated() {
+    $manager1 = ContactFabricator::fabricateWithEmail([
+      'first_name' => 'Manager1', 'last_name' => 'Manager1'], 'manager1@dummysite.com'
+    );
+
+    $leaveContact = ContactFabricator::fabricateWithEmail([
+      'first_name' => 'Staff1', 'last_name' => 'Staff1'], 'staffmember@dummysite.com'
+    );
+
+    $this->setLeaveApproverRelationshipTypes(['has Leaves Approved By']);
```

Comment:
Do we really need to specify the relationship name? Can't we use the default one?